addons/web/static/src/search/search_bar/search_bar.xml

```diff
@@ -14,13 +14,14 @@
                 <!-- :hover overlay -->
                 <div class="position-absolute start-0 top-0 bottom-0 end-0 bg-view border rounded-2 shadow opacity-0 opacity-100-hover"/>
 
-                <div class="o_searchview_facet_label position-relative rounded-start-2 px-1"
+                <div class="o_searchview_facet_label position-relative rounded-start-2 px-1 rounded-end-0 p-0"
                     t-on-click="(ev) => this.onFacetLabelClick(ev.target, facet)"
                     t-att-role="facet.domain ? 'button' : 'img'"
                     t-att-class="{
-                        'text-bg-action d-flex align-items-center': facet.type == 'groupBy' || facet.type == 'comparison',
-                        'btn btn-primary rounded-end-0 p-0': facet.type == 'field' || facet.type == 'filter',
-                        'btn btn-favourite rounded-end-0 p-0': facet.type == 'favorite'
+                        'text-bg-action': facet.type == 'groupBy' || facet.type == 'comparison',
```

Comment:
comparison style change. Why?

Reply:
removed useless styling (we only need align-items-center because of d-flex and d-flex is useless)